.circleci/config.yml

```diff
@@ -12,5 +12,6 @@ jobs:
       - run:
           command: |
             python3 -m venv venv
-            source venv/bin/activate
-            python3 setup.py test
\ No newline at end of file
+            . venv/bin/activate
+            pip install --global-option=build_ext --global-option="-I/usr/include/gdal" "GDAL <= `gdal-config --version`"
```

Comment:
@simplywondrous - I think the docker image gives us access to the C/C++ GDAL library. Then we need this line to install the GDAL python package to run the C/C++ library in our python code. Do you think that's correct?

Reply:
I thought installing the image meant GDAL would already be installed as well, but I'm not completely certain. I did test the code with CircleCI first and it worked then so I'm not sure what's up, but yeah let's revert it for now and try again later.